configure.ac

```diff
@@ -377,6 +383,20 @@ AC_DEFINE([NETDATA_WITH_UUID], [1], [uuid usability])
 OPTIONAL_UUID_CFLAGS="${UUID_CFLAGS}"
 OPTIONAL_UUID_LIBS="${UUID_LIBS}"
 
+# -----------------------------------------------------------------------------
+# WolfSSL Cryptography and SSL/TLS Toolkit
+
+AC_CHECK_LIB(
+    [wolfssl],
+    [wolfSSL_Init],
+    [WOLFSSL_LIBS="-lwolfssl"]
+)
+
+# the wolfSSL_X509_VERIFY_PARAM_set1_host is not recognize by libwebsockets, this is the motive we are searching
+
+test -z "${WOLFSSL_LIBS}" || \
+    AC_DEFINE([HAVE_CRYPTO], [1], [libwolfssl availability])
+
```

Comment:
If you'd like to simplify the following code changes, I'd suggest calling `AC_MSG_ERROR` when `"${enable_wolfssl}" = "yes"` and `${WOLFSSL_LIBS}` is empty.